### DISCUSSION TITLE
Running scripts manually?

### DISCUSSION THREAD

**Discussion by tyriekfv** [RandomNinjaAtk/arr-scripts#99] [Q&A] [answered]
I removed my extended folder as I noticed a script "metadatapostprocessbash" was no longer in use.  There may have been others that I wasnt sure about so I wanted to start new.  I removed all the connections from lidarr, and redownloaded to setup script.  Now autosetup isnt running.  I get a message at the start saying its waiting for a response from lidarr.  Then the other scripts kick in namely audio, queuecleaner, etc, but autosetup never completes. so it never adds the things back to connections or adjusts naming etc.

**Comment by tyriekfv** [comment]
2023-10-24 02:44:51 :: AutoConfig :: 3.1 :: Lidarr is not ready, sleeping until valid response...

Thats the message

**Reply by RandomNinjaAtk** [reply]
You get that message when Lidarr's API isn't responding, did you verify lidarr is actually running....?

Edit:
I have validated that it is working in my own setup... So it's most likely a user-error type of situation...

**Reply by tyriekfv** [reply]
Im not exactly sure what you mean.  Lidarr is running the other scripts are running just fine.  The autoconfig is the only one that is not working.  Lidarr is up and running as you can see in the screenshot.

Can I run the autoconfig manually somehow?

[Audio-2023_10_23_08_29_PM.txt](https://github.com/RandomNinjaAtk/arr-scripts/files/13125415/Audio-2023_10_23_08_29_PM.txt)
[QueueCleaner-2023_10_24_12_59_AM.txt](https://github.com/RandomNinjaAtk/arr-scripts/files/13125423/QueueCleaner-2023_10_24_12_59_AM.txt)
[UnmappedFilesCleaner-2023_10_24_12_59_AM.txt](https://github.com/RandomNinjaAtk/arr-scripts/files/13125429/UnmappedFilesCleaner-2023_10_24_12_59_AM.txt)
[AutoConfig-2023_10_24_03_29_AM.txt](https://github.com/RandomNinjaAtk/arr-scripts/files/13125432/AutoConfig-2023_10_24_03_29_AM.txt)
![lidarrerror](https://github.com/RandomNinjaAtk/arr-scripts/assets/36649530/92247d4d-4e31-49ca-8c07-7253473680fb)

**Reply by RandomNinjaAtk** [reply]
Like this, from container CLI:
```
root@207449a1a706:/# bash /custom-services.d/AutoConfig 
2023-10-24 15:48:46 :: AutoConfig :: 3.1 :: Configuring Lidarr Media Management Settings
2023-10-24 15:48:46 :: AutoConfig :: 3.1 :: Configuring Lidarr Metadata ConsumerSettings
2023-10-24 15:48:46 :: AutoConfig :: 3.1 :: Configuring Lidarr Metadata Provider Settings
2023-10-24 15:48:46 :: AutoConfig :: 3.1 :: Configuring Lidarr Custom Scripts
2023-10-24 15:48:46 :: AutoConfig :: 3.1 :: PlexNotify.bash Already added to Lidarr custom scripts
2023-10-24 15:48:46 :: AutoConfig :: 3.1 :: LyricExtractor.bash Already added to Lidarr custom scripts
2023-10-24 15:48:46 :: AutoConfig :: 3.1 :: ArtworkExtractor.bash Already added to Lidarr custom scripts
2023-10-24 15:48:46 :: AutoConfig :: 3.1 :: BeetsTagger.bash Already added to Lidarr custom scripts
2023-10-24 15:48:46 :: AutoConfig :: 3.1 :: Configuring Lidarr UI Settings
2023-10-24 15:48:46 :: AutoConfig :: 3.1 :: Configuring Lidarr Standard Metadata Profile
2023-10-24 15:48:46 :: AutoConfig :: 3.1 :: Configuring Lidarr Track Naming Settings
```

**Reply by tyriekfv** [reply]
these are the blank spaces in the file complaining as command not found

root@ee482920b0ef:/# bash /custom-services.d/AutoConfig 
/config/extended.conf: line 2: $'\r': command not found
/config/extended.conf: line 6: $'\r': command not found
/config/extended.conf: line 13: $'\r': command not found
/config/extended.conf: line 20: $'\r': command not found
/config/extended.conf: line 29: $'\r': command not found
/config/extended.conf: line 36: $'\r': command not found
/config/extended.conf: line 44: $'\r': command not found
/config/extended.conf: line 50: $'\r': command not found
/config/extended.conf: line 55: $'\r': command not found
/config/extended.conf: line 63: $'\r': command not found
/config/extended.conf: line 68: $'\r': command not found

**Reply by tyriekfv** [reply]
Ok so I switched out for a backup config file and it worked fine.  So something is going on with my extended.conf file that I dont see.

**Reply by githubusr80** [reply]
You most likely edited the file using a windows based text editor and ended up changing the line endings to an incompatible format.

You need to use a proper text editor to maintain the correct line endings. Google it for more info.

**Comment by tyriekfv** [comment]
> You most likely edited the file using a windows based text editor and ended up changing the line endings to an incompatible format.
> 
> You need to use a proper text editor to maintain the correct line endings. Google it for more info.

I use notepad++

**Reply by githubusr80** [reply]
Make sure the line endings are set to Unix then.

**Reply by tyriekfv** [reply, marked as answer]
I think what  you mentioned is what happened.  When I downloaded the new file and opened it, I DID open it in regular notepad then saved and opened it in Notepad++.  I went back to an older one without an issue.  Then downloaded the file again, edited it with my changes and that one works as well.
